std/didexchange/v0/invitation_impl.go

```diff
@@ -111,7 +109,7 @@ func (m *invitationImpl) RoutingKeys() []string {
 	return m.Invitation.Services()[0].RoutingKeysAsB58()
 }
 
-func (m *invitationImpl) Verify(c crypto.Crypto, keyManager kms.KeyManager) error {
+func (m *invitationImpl) Verify(DID core.DID) error {
 	return nil
```

Comment:
Is it safe to return just nil?

Reply:
Yes, there is nothing to verify for this message type.